projects/services-ogc/src/lib/wps/test/wpsclient.spec.ts

```diff
@@ -11,7 +9,11 @@ import * as OWS_2_0_Factory from 'ogc-schemas/lib/OWS_2_0'; const OWS_2_0 = OWS_
 import * as WPS_1_0_0_Factory from 'ogc-schemas/lib/WPS_1_0_0'; const WPS_1_0_0 = WPS_1_0_0_Factory.WPS_1_0_0;
 import * as WPS_2_0_Factory from 'ogc-schemas/lib/WPS_2_0'; const WPS_2_0 = WPS_2_0_Factory.WPS_2_0;
 import { Jsonix } from '@michaellangbein/jsonix';
-import { WpsData, WpsDataDescription, WpsVerion, isWpsState } from './wps_datatypes';
+import { FakeWpsServer } from './fake_wps_server';
+import { PollableServer } from './pollable_server';
+import { pollUntil } from '../utils/polling';
+import { WpsClient } from '../wpsclient';
+import { isWpsState, WpsData, WpsDataDescription, WpsVerion } from '../wps_datatypes';
 
 class MyXhrFactory extends XhrFactory {
```

Comment:
Can you maybe rename this? I think we don't should prefix things with **my**

Reply:
Has been removed!